tests/automation_framework/src/utilities/worker_utilities.py

```diff
@@ -245,7 +247,7 @@ def config_data_update(input, key, value):
 
 def read_config(config_file, test_name):
```

Comment:
Hope the modifications to the yaml file reading has been tested. Rest it looks okay.

Reply:
Yes. Whole test suite is run with sdk and listener.